src/js/LottieView.js

```diff
@@ -6,11 +6,11 @@ import {
   View,
   Platform,
   StyleSheet,
-  ViewPropTypes,
   requireNativeComponent,
   NativeModules,
   processColor,
 } from 'react-native';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
```

Comment:
@emilioicai seems this new import is breaking the latest release as the dependency lib "deprecated-react-native-prop-types" is not included in package.json

Reply:
sorry for that, v5.1.1 was just released with a fix for that